pkg/cloud/scope/machine.go

```diff
@@ -343,6 +343,17 @@ func (m *MachineScope) IsEKSManaged() bool {
 	return m.InfraCluster.InfraCluster().GetObjectKind().GroupVersionKind().Kind == "AWSManagedControlPlane"
 }
 
+func (m *MachineScope) IsExternallyManaged() bool {
```

Comment:
we can actually use https://github.com/kubernetes-sigs/cluster-api/commit/3065a926259f682f65cb8331b5f2543b270882a8#diff-e25fb86695a8022fa1611100bd244f9b4185f28629d19830e83d18b4fa2ce710R34 here